orchestrator/careplancontributor/ehr/notifier.go

```diff
@@ -0,0 +1,373 @@
+package ehr
+
+import (
+	"encoding/json"
+	"fmt"
+	fhirclient "github.com/SanteonNL/go-fhir-client"
+	"github.com/SanteonNL/orca/orchestrator/lib/coolfhir"
+	"github.com/google/uuid"
+	"github.com/rs/zerolog/log"
+	"github.com/zorgbijjou/golang-fhir-models/fhir-models/fhir"
+	"slices"
+	"strings"
+)
+
+type Notifier interface {
+	NotifyTaskAccepted(cpsClient fhirclient.Client, task *fhir.Task) error
+}
+
+type notifier struct {
+	kafkaClient KafkaClient
+}
+
+func NewNotifier(kafkaClient KafkaClient) Notifier {
+	return &notifier{kafkaClient}
+}
+
+// BundleSet represents a set of FHIR bundles associated with a task.
+type BundleSet struct {
+	Id      string
+	task    string
+	Bundles []fhir.Bundle `json:"bundles"`
```

Comment:
I thought we'd flatten it to 1 Bundle?

Reply:
Nope, that would not educate towards fetching the bundles later on. If we start modifying the stuff we get from the FHIR resource service we go down an undesirable path om modifying data.